hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerRatisServer.java

```diff
@@ -246,11 +250,16 @@ public OMResponse submitRequest(OMRequest omRequest) throws ServiceException {
     // In prepare mode, only prepare and cancel requests are allowed to go
     // through.
     if (ozoneManager.getPrepareState().requestAllowed(omRequest.getCmdType())) {
-      RaftClientRequest raftClientRequest =
-          createWriteRaftClientRequest(omRequest);
-      RaftClientReply raftClientReply = submitRequestToRatis(raftClientRequest);
-
-      return processReply(omRequest, raftClientReply);
+      RaftClientRequest raftClientRequest = captureLatencyNs(
+          perfMetrics.getConvertRatisRequestLatencyNs(),
+          () -> createWriteRaftClientRequest(omRequest));
```

Comment:
Let's use the same name?  i.e.  when the method  name is ` createWriteRaftClientRequest`, the metric name should be  `getCreateWriteRaftClientRequestLatencyNs()`.

If the existing method name is not good, we may change it as well.

Reply:
Also, let's create new methods to hide the metric layer as below.  The code will be more readable.
```java
@@ -414,13 +418,19 @@ public void removeRaftPeer(OMNodeDetails omNodeDetails) {
     LOG.info("{}: Removed OM {} from Ratis Peers list.", this, removeNodeID);
   }
 
+  private RaftClientRequest createWriteRaftClientRequest(OMRequest omRequest) {
+    return captureLatencyNs(
+        perfMetrics.getCreateWriteRaftClientRequestLatencyNs(),
+        () -> createWriteRaftClientRequestImpl(omRequest));
+  }
+
   /**
    * Create Write RaftClient request from OMRequest.
    * @param omRequest
    * @return RaftClientRequest - Raft Client request which is submitted to
    * ratis server.
    */
-  private RaftClientRequest createWriteRaftClientRequest(OMRequest omRequest) {
+  private RaftClientRequest createWriteRaftClientRequestImpl(OMRequest omRequest) {
     if (!ozoneManager.isTestSecureOmFlag()) {
       Preconditions.checkArgument(Server.getClientId() != DUMMY_CLIENT_ID);
       Preconditions.checkArgument(Server.getCallId() != INVALID_CALL_ID);
```

Reply:
> Let's use the same name? i.e. when the method name is  createWriteRaftClientRequest, the metric name should be  getCreateWriteRaftClientRequestLatencyNs().

I agree with maintaining consistency by using the same name for methods and metrics.

Reply:
Agree. Updated.